packages/ListBox/stories/examples/Showcase.js

```diff
@@ -1,15 +1,19 @@
 import React from "react";
 import { action } from "@storybook/addon-actions";
-import { boolean, number, text } from "@storybook/addon-knobs";
+import { boolean, number, select, text } from "@storybook/addon-knobs";
 import CodeViewer from "storybook/components/CodeViewer";
 import StoryHeading from "storybook/components/StoryHeading";
 import * as characters from "../fixtures/characters";
 import ListBox from "../../src";
 
 const getKnobs = () => ({
+  size: select("size", Object.values(ListBox.types.size), ListBox.types.size.MEDIUM),
   height: number("height", 200),
+  hasError: boolean("hasError", false),
   isDisabled: boolean("isDisabled", false),
-  placeholder: text("placeholder", "Select..."),
+  isInline: boolean("isInline", false),
+  isReadOnly: boolean("isReadOnly", false),
+  placeholder: text("placeholder", ""),
```

Comment:
With the new approach, these props can now be controlled with the Storybook knobs.  Additionally, if the consuming app changes their values on a re-render, after the component has mounted, the component will be updated.